abstract-account-keys/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "mc-abstract-account-keys"
```

Comment:
Optional/Nit: `abstract` seems out of place in Rust. How about `mc-account-key-traits`?